internal/pkg/manifest/backend_svc.go

```diff
@@ -41,7 +41,8 @@ type BackendServiceConfig struct {
 // BackendServiceProps represents the configuration needed to create a backend service.
 type BackendServiceProps struct {
 	WorkloadProps
-	Port        uint16
+	Ports       []uint16
+	Path        string               // Optional path if multiple ports are exposed.
```

Comment:
does http field on a backend service lead to internal alb being created? I'm confused how this works.

Reply:
Yes you are right. http field in BS creates internal ALB. And hence we are allowing customer to expose multiple ports via internal ALB.